extension-style/src/main/java/com/mapbox/maps/extension/style/expressions/generated/Expression.kt

```diff
@@ -2301,6 +2305,139 @@ class Expression : Value {
     fun cos(block: ExpressionBuilder.() -> Unit): Expression =
       ExpressionBuilder("cos").apply(block).build()
 
+    fun BoundingBox.toValue(): Value {
```

Comment:
Previously geojson was [serialized](https://github.com/mapbox/mapbox-java/blob/a86a6148b87ff3e4a9bd1096eaed5715b80db5e1/services-geojson/src/main/java/com/mapbox/geojson/BaseGeometryTypeAdapter.java) to json, then parsed back from string.